lib/call-cabal-project-to-nix.nix

```diff
@@ -401,7 +401,7 @@ let
         exit 1
       ''}
     else
-      cp -r ${cleanedSource}/* .
```

Comment:
The `${cleanedSource}/*` glob only matches non-hidden children of `${cleanedSource}`, whereas rsync with `${cleanedSource}/` will pick up _all_ children.